specification/network/resource-manager/readme.md

```diff
@@ -1012,6 +1012,48 @@ directive:
   - suppress: RequiredPropertiesMissingInResourceModel
     from: webapplicationfirewall.json
     reason: name, id and type properties are inherited from the upper level
+  - suppress: PostOperationIdContainsUrlVerb
```

Comment:
@EvgenyAgafonchikov were these suppressions approved?

Reply:
+@ravbhatnagar, could you please review?

Mostly this suppresses warnings for operations using custom verbs. These changes are already released in SDKs and related warnings currently overcrowd validator's output that may lead to missing of the new entries.